demos/python_demos/multi_camera_multi_person_tracking/tools/run_history_visualize.py

```diff
@@ -0,0 +1,249 @@
+"""
+ Copyright (c) 2019 Intel Corporation
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+import argparse
+import copy
+import json
+import logging as log
+
+import cv2 as cv
+import motmetrics as mm
+import numpy as np
+from tqdm import tqdm
+
+from tools.run_evaluate import read_gt_tracks, get_detections_from_tracks
+from utils.misc import check_pressed_keys, set_log_config
+from utils.video import MulticamCapture
+from utils.visualization import visualize_multicam_detections, plot_timeline, get_target_size
+
+set_log_config()
+
+
+def find_max_id(all_tracks):
+    def find_max(tracks):
+        max_id = 0
+        for track in tracks:
+            if track['id'] > max_id:
+                max_id = track['id']
+        return max_id
+
+    output = []
+    for cam_tracks in all_tracks:
+        output.append(find_max(cam_tracks))
+    return max(output)
```

Comment:
Isn't all of this basically `return max(track['id'] for cam_tracks in all_tracks for track in cam_tracks)`?

Reply:
Fixed.